sample_files/.travis.yml

```diff
@@ -11,9 +11,21 @@ addons:
 
 env:
   global:
-  - VERSION="8.0" TESTS="0" LINT_CHECK="0"
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
```

Comment:
Thus you would have a noisy `TESTS="0"` on transifex and lint check lines or anything else we would add. We could consider all jobs should be tests unless it isn't. Or that it needs to always be defined.
It's a matter of taste.